### PR TITLE
Fix: Make Scan_Realtime product available to UI.

### PR DESCRIPTION
**Pull request by elliottprogrammer**
### Changes proposed in this Pull Request

Fixes 1164141197617539-as-1199554500149274 (Scan section shows upsell when user has Scan Realtime)

Scan Realtime is not available to purchase as an individual product at this time, however if the product is added through the Store Admin then Calypso blue and green was not recognizing it at all and consequentially it was not showing as a purchased product on the plans/pricing page and also the Scan upsell **was** showing when going to Jetpack --> Scan.

This PR makes Scan Realtime product available to the Calypso UI for if it is added through the Store Admin or for when we may want to make it available for purchase as an individual product in the future.  It is still not available in the UI for purchase from the public, but it **is** available in the UI if it has already been purchased (through the Store Admin).
 
### Testing instructions

- Checkout and run this PR in calypso blue & green concurrently.
- Select a Jetpack site that doesn't have Scan yet
- Subscribe to Scan Realtime through the Store Admin
- Visit Plan --> Billing page (`http://calypso.localhost:3000/purchases/subscriptions/:site`) and verify that "Scan Realtime" is showing as a purchased product. (see Fig.1)
- Visit `/scan/:site` in Calypso blue & green and verify that the Scan component is showing and **not** the Scan Upsell. (See fig.2)
- Visit the `/plans/:site` in Calypso blue & green and verify that "Scan Realtime" product card is showing as a purchased product (with "Manage Subscription" button) (See Fig.3)
    - Note: There is not a correct price set for Scan Realtime because it has not been set for purchase to the public
- Verify there are no errors in the console.

**Fig.1**
Markup 2021-01-19 at 14 40 32

**Fig.2**
Markup 2021-01-19 at 14 42 57

**Fig.3**
Markup 2021-01-19 at 14 44 15

### CODE REVIEW

File: client/lib/products-values/translations.js [hunk 1]
```diff
@@ -86,6 +96,21 @@ export const getJetpackProductsDisplayNames = () => {
 			i5: translate( 'Site Search' ),
 		}[ currentCROvariant ] || translate( 'Jetpack Search' );
 	const scan = currentCROvariant === 'i5' ? translate( 'Scan' ) : translate( 'Jetpack Scan' );
+	const scanRealtime = (
+		<>
+			{ currentCROvariant === 'i5'
+				? translate( 'Scan {{em}}Real-Time{{/em}}', {
```
Comment (a8ci18n) [inline client/lib/products-values/translations.js:119]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Scan {{em}}Real-time{{/em}}' )` *ES Score: 12*

File: client/lib/products-values/translations.js [hunk 2]
```diff
@@ -86,6 +96,21 @@ export const getJetpackProductsDisplayNames = () => {
...
+						components: {
+							em: <em />,
+						},
+				  } )
+				: translate( 'Jetpack Scan {{em}}Real-Time{{/em}}', {
```
Comment (a8ci18n) [inline client/lib/products-values/translations.js:124]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Scan {{em}}Real-time{{/em}}' )` *ES Score: 9*

**Review by monsieur-z — APPROVED** [review]
LGTM!

**Comment by rcanepa** [comment]
Do we need to update the plugin to make it support this product too?

**Comment by monsieur-z** [comment]
> Do we need to update the plugin to make it support this product too?

It's showing Jetpack Free when you own Scan Real-time, so I'd say yes.

**Comment by elliottprogrammer** [comment]
> > Do we need to update the plugin to make it support this product too?
> 
> It's showing Jetpack Free when you own Scan Real-time, so I'd say yes.

Even so, It's probably ok to merge this, right?  And then I'll take a look in the plugin to investigate what it will take to add support for Scan Realtime. Does that sound like the correct next steps?

**Comment by rcanepa** [comment]
> Even so, It's probably ok to merge this, right? And then I'll take a look in the plugin to investigate what it will take to add support for Scan Realtime. Does that sound like the correct next steps?

Yes, that sounds correct to me. 🙂

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5468215

Thank you @elliottprogrammer for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5468215).

**Pull request merged**